testsuite/dependencies/resize/resize4.cc

```diff
@@ -0,0 +1,22 @@
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE libstorage
+
+#include <boost/test/unit_test.hpp>
+
+#include "storage/Utils/Logger.h"
+#include "testsuite/helpers/TsCmp.h"
+
+
+using namespace storage;
+
+
+BOOST_AUTO_TEST_CASE(dependencies)
+{
+    set_logger(get_stdout_logger());
+
+    //
```

Comment:
NP: it would be good an explanation as in other resize tests. Otherwise, I would simply remove the comment.